crates/invoker-impl/src/lib.rs

```diff
@@ -1090,6 +1093,15 @@ where
             .remove_invocation_with_epoch(partition, &invocation_id, invocation_epoch)
         {
             debug_assert_eq!(invocation_epoch, ism.invocation_epoch);
+
+            if self.is_service_down_error(&error.kind) {
+                let deployment_id = error
+                    .deployment_id
+                    .map(|id| Cow::Owned(id.to_string()))
+                    .unwrap_or_else(|| Cow::Borrowed("unknown"));
+                counter!(INVOKER_DEPLOYMENT_UNREACHABLE_ERRORS, "service" => ism.invocation_target.service_name().to_string(), "deployment" => deployment_id).increment(1);
```

Comment:
Is this something we can already detect with the new fttb metric, or the invocation task status metric or does it require a new metric?